### DISCUSSION TITLE
Diatone Mamba Basic f722 MK3 using I2C

### DISCUSSION THREAD

**Discussion by kbarthen** [iNavFlight/inav#8140] [General]
Hi,
I wondered if anyone has had any luck with the Mamba Basic f722 MK3 FC? Using inav 4.1 I flashed with the mamba f722 i2c target and do not get gyro or accel? Baro, Mag light up but no luck with basic gyro/accel sensors. I can flash back to Betaflight and works / shows fine. I am curious if this is something anyone has experienced before, and are there any possible work-arounds, or possibly updates coming down with inav 5.0 that would help?

Note the plan was to use this board because it offered a baro with a Matek GPS and the cool nav features of inav. But for now just a paperweight. :-0

Any help / guidance is appreciated.

Thanks.

**Comment by DzikuVx** [comment]
Please check if 5.0 RC3 is working. There were gyro updates for this target lately

**Comment by jason1973x** [comment]
i have the same board it has worked in tests but pos hold is not showing up in modes and keeps telling me to update my firmware on 5.0.0 rc3?
